scripts/sync-kube-mixin.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# make a temporary dir to work in
+TMPDIR=$(mktemp -d -t 'tmp.XXXXXXXXXX')
+trap 'rm -f $TMPDIR' EXIT
```

Comment:
Do you want to `rm -f` or `rm -rf` at exit, like at the end of the script?
Maybe that should be factorized in a common function?

Reply:
I mean, same code for same usage in 2 places, I thought it deserves a function. The syntax difference just proved the point :grin: